frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf

```diff
@@ -70,7 +70,7 @@
                 '<select class="toplabelledInput" ' +
                 'id="verificationChangeSelectFor<sw360:out value='${vulnerability.externalId}'/>" ' +
                 'name="<portlet:namespace/><%=VerificationState.class%>" ' +
-                'onchange="changeVerification(this, `<sw360:out value='${vulnerability.externalId}'/>`,`${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}`)" ' +
+                "onchange='changeVerification(this, &quot;<sw360:out value='${vulnerability.externalId}'/>&quot;,&quot;${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}&quot;)' " +
```

Comment:
This does not work.

I have created a fix for this at https://github.com/bsinno/sw360portal/tree/fix/minifierErrors. You can cherry-pick the commit from where if you want.